src/Makefile.qt.include

```diff
@@ -378,6 +378,20 @@ bitcoin_qt_clean: FORCE
 
 bitcoin_qt : qt/bitcoin-qt$(EXEEXT)
 
+APK_LIB_DIR = qt/android/libs/$(ANDROID_ARCH)
```

Comment:
Shouldn't the content of the `qt/android` directory be a subject of the cleanup in `make clean` or `make distclean`?

Reply:
It should :+1: